.github/workflows/docker-dev-build.yml

```diff
@@ -56,10 +56,10 @@ jobs:
         uses: actions/checkout@v2
       - name: 🐳 Docker meta
         id: docker_meta
-        uses: crazy-max/ghaction-docker-meta@v1
+        uses: docker/metadata-action@v3
         with:
           images: ${{ secrets.DOCKERHUB_REPO }}
-          label-custom: |
+          labels: |
             org.opencontainers.image.title=Gladys Assitant Preprod Image
```

Comment:
Not related to this PR, but there is a typo here, should be: Gladys Assistant Preprod Image